Maddori.Apple/Maddori.Apple/Screen/AddFeedbackContent/AddFeedbackContentViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  AddFeedbackContentViewController.swift
+//  Maddori.Apple
+//
+//  Created by 김유나 on 2022/10/19.
+//
+
+import UIKit
+
+import SnapKit
+
+final class AddFeedbackContentViewController: BaseViewController {
+    
+    // MARK: - property
+
+    private let backButton = BackButton()
+    private let closeButton = CloseButton()
```

Comment:
p2;

요 두 버튼들이 눌러지지가 않아서 
```suggestion
    private let backButton = BackButton(type: .system)
    private let closeButton = CloseButton(type: .system)
```
요 친구들을 추가해주실 수 있나요오??

Reply:
[0258ede](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/27/commits/0258ede9052a762faff999a6bc6d04c00a59695e)
오 이래서 작동 안 하는 거였군요 ! 수정해서 반영해두었습니다 :)